shared/metrics/src/main/java/io/pravega/shared/MetricsNames.java

```diff
@@ -334,6 +348,7 @@ public final class MetricsNames {
     public static final String ABORT_TRANSACTION_FAILED = PREFIX + "controller.transactions.abort_failed";                              // Counter, Per-stream Counter, Per-transaction Counter
     public static final String OPEN_TRANSACTIONS = PREFIX + "controller.transactions.opened";                                           // Per-stream Gauge
     public static final String TIMEDOUT_TRANSACTIONS = PREFIX + "controller.transactions.timedout";                                     // Per-stream Counter
+    public static final String CONTROLLER_EVENT_PROCESSOR_COMMIT_TRANSACTION_LATENCY = PREFIX + "controller.eventProcessors.transactions.commit.write_latency_ms";      // Histogram
```

Comment:
Why this one does not go up with all the other `CONTROLLER_EVENT_PROCESSOR` metrics?

Reply:
This is Transaction related so put it under
 // Transaction request Operations
let  me move it with other CONTROLLER_EVENT_PROCESSOR